creativecloud/scripts/scripts.js

```diff
@@ -161,6 +156,27 @@ const CONFIG = {
 
 const miloLibs = setLibs(LIBS);
 
+(async function loadScript() {
```

Comment:
Can we a more specific function name to convey that this is loading changeBG script